<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diagnosis</class>
 <widget class="QWidget" name="diagnosis">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>诊断编辑器</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARDS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>肾功能衰竭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>肝功能衰竭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>心力衰竭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>呼吸衰竭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>循环衰竭</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>胰腺炎</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>肺炎</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>加入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>270</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>写入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>270</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>离开</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>diagnosis</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
